config/earlyapp_gst.target

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=earlyapp target
+Wants=earlyapp_gst.service
+After=earlyapp_gst.service
+
+# Delay some heavier services to speed up earlyapp loading
+Before=systemd-udevd.service systemd-udev-trigger.service systemd-modules-load.service
+
+[Install]
+WantedBy=basic.target
+Also=earlyapp_resume.service
```

Comment:
Confirm whether we can keep using the earlyapp_resume.service for the GStreamer environment.

Reply:
Yes, its still applicable in generic platform